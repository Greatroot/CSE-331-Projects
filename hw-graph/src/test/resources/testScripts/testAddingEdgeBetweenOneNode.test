#Test adding an edge pointing from and to the same node | should add the edge.

CreateGraph graph
AddNode graph n1
AddEdge graph n1 n1 e1
