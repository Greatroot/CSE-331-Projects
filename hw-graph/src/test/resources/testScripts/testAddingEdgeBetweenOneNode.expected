#Test adding an edge pointing from and to the same node | should add the edge.

created graph graph
added node n1 to graph
added edge e1 from n1 to n1 in graph
